usr/src/cmd/zdb/zdb.c

```diff
@@ -1111,7 +1116,7 @@ dump_history(spa_t *spa)
 	umem_free(buf, SPA_MAXBLOCKSIZE);
 
 	(void) printf("\nHistory:\n");
-	for (int i = 0; i < num; i++) {
+	for (unsigned i = 0; i < num; i++) {
```

Comment:
i'd like to see fixed compiler warnings :)
i do checks by gcc5/gcc6 on my env with DilOS builds and i have fixed more warnings, but not upstreamed it yet ...
i have tried upstream couple of updates with cmd/zfs & cmd/zpool tools, but have to works on others too and i have no time right now ...
i'd like to test updates on my tree with builds by gcc5/6 on Intel and still with gcc44 on sparc.